log-example/src/main/resources/log4j-async.xml

```diff
@@ -7,7 +7,7 @@
         </layout>
     </appender>
     <appender name="http" class="io.phdata.pulse.log.HttpAppender">
-        <param name="Address" value="http://edge2.valhalla.phdata.io:9001/log?application=pulse-test-100"/>
+        <param name="Address" value="http://${ZK_EDGE}:9001/log?application=pulse-test-100"/>
```

Comment:
This is the log collector host, you can just put something like 'log_collector_hostname'